backend/models/playlist.js

```diff
@@ -61,14 +66,13 @@ class Playlist {
 
     //from 5 songs, get recommendations
     static async getRec(seedSongs, token) {
-        //TODO: sometimes errors out here: "cannot read track"
         let str = seedSongs[0].track.id
         for (let i=1; i < seedSongs.length-1; i++) {
             str.concat(`,${seedSongs[i].track.id}`)
 
         }
 
-        const response = await axios.get(`https://api.spotify.com/v1/recommendations?min_popularity=50&limit=8&seed_tracks=${str}`, {
+        const response = await axios.get(`https://api.spotify.com/v1/recommendations?min_popularity=50&limit=12&seed_tracks=${str}`, {
```

Comment:
all these numbers here should ideally by constants instead of string literals. This way it is clear what they represent and easy for another developer to update in the future.